Excel Exporter/Importer Wizard creates "Excel IO Failed Exception"

Create a project and select Export -> Virtual Satellite -> Excel.

The following exception is then thrown 

![grafik](https://user-images.githubusercontent.com/48356419/94141513-255d8e00-fe6d-11ea-9e43-29b9a0d37c3b.png)

`org.eclipse.core.runtime.CoreException: Plug-in "de.dlr.sc.virsat.model.extension.funcelectrical" was unable to instantiate class "de.dlr.sc.virsat.model.extension.funcelectrical.excel.exporter.FuncElecExporter".
	at org.eclipse.core.internal.registry.osgi.RegistryStrategyOSGI.throwException(RegistryStrategyOSGI.java:212)
	at org.eclipse.core.internal.registry.osgi.RegistryStrategyOSGI.createExecutableExtension(RegistryStrategyOSGI.java:206)
	at org.eclipse.core.internal.registry.ExtensionRegistry.createExecutableExtension(ExtensionRegistry.java:923)
	at org.eclipse.core.internal.registry.ConfigurationElement.createExecutableExtension(ConfigurationElement.java:246)
	at org.eclipse.core.internal.registry.ConfigurationElementHandle.createExecutableExtension(ConfigurationElementHandle.java:63)
	at de.dlr.sc.virsat.excel.exporter.ExcelExporter.canExport(ExcelExporter.java:68)
	at de.dlr.sc.virsat.excel.ui.wizards.ExportWizard.canFinish(ExportWizard.java:119)
	at org.eclipse.jface.wizard.WizardDialog.updateButtons(WizardDialog.java:1331)
	at org.eclipse.jface.wizard.WizardDialog.update(WizardDialog.java:1322)
	at org.eclipse.jface.wizard.WizardDialog.updateForPage(WizardDialog.java:1257)
	at org.eclipse.jface.wizard.WizardDialog.lambda$3(WizardDialog.java:1222)
	at org.eclipse.swt.custom.BusyIndicator.showWhile(BusyIndicator.java:72)
	at org.eclipse.jface.wizard.WizardDialog.showPage(WizardDialog.java:1222)
	at org.eclipse.jface.wizard.WizardDialog.nextPressed(WizardDialog.java:917)
	at org.eclipse.jface.wizard.WizardDialog.buttonPressed(WizardDialog.java:468)
	at org.eclipse.jface.dialogs.Dialog.lambda$0(Dialog.java:619)
	at org.eclipse.swt.events.SelectionListener$1.widgetSelected(SelectionListener.java:84)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(TypedListener.java:252)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:89)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4213)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1037)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Display.java:4030)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3630)
	at org.eclipse.jface.window.Window.runEventLoop(Window.java:823)
	at org.eclipse.jface.window.Window.open(Window.java:799)
	at org.eclipse.ui.internal.handlers.WizardHandler$Export.executeHandler(WizardHandler.java:103)
	at org.eclipse.ui.internal.handlers.WizardHandler.execute(WizardHandler.java:280)
	at org.eclipse.ui.internal.handlers.HandlerProxy.execute(HandlerProxy.java:283)
	at org.eclipse.ui.internal.handlers.E4HandlerProxy.execute(E4HandlerProxy.java:97)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.e4.core.internal.di.MethodRequestor.execute(MethodRequestor.java:58)
	at org.eclipse.e4.core.internal.di.InjectorImpl.invokeUsingClass(InjectorImpl.java:319)
	at org.eclipse.e4.core.internal.di.InjectorImpl.invoke(InjectorImpl.java:253)
	at org.eclipse.e4.core.contexts.ContextInjectionFactory.invoke(ContextInjectionFactory.java:173)
	at org.eclipse.e4.core.commands.internal.HandlerServiceHandler.execute(HandlerServiceHandler.java:156)
	at org.eclipse.core.commands.Command.executeWithChecks(Command.java:488)
	at org.eclipse.core.commands.ParameterizedCommand.executeWithChecks(ParameterizedCommand.java:487)
	at org.eclipse.e4.core.commands.internal.HandlerServiceImpl.executeHandler(HandlerServiceImpl.java:213)
	at org.eclipse.ui.internal.handlers.LegacyHandlerService.executeCommand(LegacyHandlerService.java:389)
	at org.eclipse.ui.internal.actions.CommandAction.runWithEvent(CommandAction.java:142)
	at org.eclipse.jface.action.ActionContributionItem.handleWidgetSelection(ActionContributionItem.java:579)
	at org.eclipse.jface.action.ActionContributionItem.lambda$4(ActionContributionItem.java:413)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:89)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4213)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1037)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Display.java:4030)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3630)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1158)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1047)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:658)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:557)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:154)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:150)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:137)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:107)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:401)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:657)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:594)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1447)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1420)
Caused by: java.lang.NoSuchMethodException: de.dlr.sc.virsat.model.extension.funcelectrical.excel.exporter.FuncElecExporter.<init>()
	at java.lang.Class.getConstructor0(Class.java:3082)
	at java.lang.Class.getDeclaredConstructor(Class.java:2178)
	at org.eclipse.core.internal.registry.osgi.RegistryStrategyOSGI.createExecutableExtension(RegistryStrategyOSGI.java:204)
	... 70 more`

--- a/de.dlr.sc.virsat.model.extension.funcelectrical/src/de/dlr/sc/virsat/model/extension/funcelectrical/excel/exporter/FuncElecExporter.java
+++ b/de.dlr.sc.virsat.model.extension.funcelectrical/src/de/dlr/sc/virsat/model/extension/funcelectrical/excel/exporter/FuncElecExporter.java
@@ -49,14 +49,8 @@ import de.dlr.sc.virsat.model.extension.ps.model.ElementRealization;
  */
 public class FuncElecExporter implements IExport {
 	
-	public FuncElecExporter(LocalDateTime localDateTime) {
-		this.localDateTime = localDateTime;
-	}
-	
-	protected LocalDateTime localDateTime;
 	private static final String DEFAULT_TEMPLATE_PATH = "/resources/ExcelExportTemplate.xlsx";
-	private ExcelExportHelper helper;
-
+	
 	private static final String[] EXPORTABLE_SEIS = {
 			ElementDefinition.class.getSimpleName(),
 			ElementConfiguration.class.getSimpleName(),
@@ -64,6 +58,17 @@ public class FuncElecExporter implements IExport {
 			ElementRealization.class.getSimpleName(),
 			ElementOccurence.class.getSimpleName()
 	};
+	
+	protected LocalDateTime localDateTime;
+	private ExcelExportHelper helper;
+	
+	public FuncElecExporter() {
+		this.localDateTime = LocalDateTime.now(); 
+	}
+	
+	public FuncElecExporter(LocalDateTime localDateTime) {
+		this.localDateTime = localDateTime;
+	}
 
 	@Override
 	public void export(EObject eObject, String path, boolean useDefaultTemplate, String templatePath) {

--- a/de.dlr.sc.virsat.model.extension.funcelectrical/src/de/dlr/sc/virsat/model/extension/funcelectrical/excel/exporter/FuncElecExporter.java
+++ b/de.dlr.sc.virsat.model.extension.funcelectrical/src/de/dlr/sc/virsat/model/extension/funcelectrical/excel/exporter/FuncElecExporter.java
@@ -63,7 +63,7 @@ public class FuncElecExporter implements IExport {
 	private ExcelExportHelper helper;
 	
 	public FuncElecExporter() {
-		this.localDateTime = LocalDateTime.now(); 
+		this(LocalDateTime.now());
 	}
 	
 	public FuncElecExporter(LocalDateTime localDateTime) {

--- a/de.dlr.sc.virsat.model.extension.statemachines/src/de/dlr/sc/virsat/model/extension/statemachines/excel/exporter/StateMachineExporter.java
+++ b/de.dlr.sc.virsat.model.extension.statemachines/src/de/dlr/sc/virsat/model/extension/statemachines/excel/exporter/StateMachineExporter.java
@@ -46,7 +46,7 @@ public class StateMachineExporter implements IExport {
 	private CategoryAssignment exportCa;
 	
 	public StateMachineExporter() {
-		this.localDateTime = LocalDateTime.now(); 
+		this(LocalDateTime.now());
 	}
 	
 	public StateMachineExporter(LocalDateTime localDateTime) {

--- a/de.dlr.sc.virsat.model.extension.statemachines/src/de/dlr/sc/virsat/model/extension/statemachines/excel/exporter/StateMachineExporter.java
+++ b/de.dlr.sc.virsat.model.extension.statemachines/src/de/dlr/sc/virsat/model/extension/statemachines/excel/exporter/StateMachineExporter.java
@@ -39,14 +39,19 @@ import de.dlr.sc.virsat.model.extension.statemachines.model.Transition;
  */
 public class StateMachineExporter implements IExport {
 	
+	private static final String DEFAULT_TEMPLATE_PATH = "/resources/StateMachineExportTemplate.xlsx";
+	
+	protected LocalDateTime localDateTime;
+	protected ExcelExportHelper helper = new ExcelExportHelper();
+	private CategoryAssignment exportCa;
+	
+	public StateMachineExporter() {
+		this.localDateTime = LocalDateTime.now(); 
+	}
+	
 	public StateMachineExporter(LocalDateTime localDateTime) {
 		this.localDateTime = localDateTime;
 	}
-	
-	protected LocalDateTime localDateTime;
-	private static final String DEFAULT_TEMPLATE_PATH = "/resources/StateMachineExportTemplate.xlsx";
-	protected ExcelExportHelper helper = new ExcelExportHelper();
-	private CategoryAssignment exportCa;
 
 	@Override
 	public void export(EObject eObject, String path, boolean useDefaultTemplate, String templatePath) {

